public/js/monitor.js

```diff
@@ -57,61 +57,81 @@ function isValidEmail(val) {
   return re.test(String(val).toLowerCase());
 }
 
-function addClass(selector, className){
+function addClass(selector, className) {
   const el = document.querySelector(selector);
-  if(el){
+  if (el) {
     el.classList.add(className);
   }
 }
 
-function removeClass(selector, className){
+function removeClass(selector, className) {
   const el = document.querySelector(selector);
-  if(el){
+  if (el) {
     el.classList.remove(className);
   }
 }
 
-function removeInvalidMessage(){
+function removeInvalidMessage() {
   removeClass("#invalid-email-message", "show");
   removeClass(".input-group","invalid");
 }
 
-function addInvalidMessage(){
+function addInvalidMessage() {
   addClass("#invalid-email-message", "show");
   addClass(".input-group", "invalid");
 }
 
 function enableBtnIfEmailValid(e) {
+  removeInvalidMessage();
   const emailBtn = document.getElementById("submit-email");
   if (isValidEmail(e.target.value)) {
       emailBtn.disabled = false;
       removeInvalidMessage();
   } else {
       emailBtn.disabled = true;
+      if(e.code === "Enter") {
+        addInvalidMessage();
+      }
   }
 }
 
 function enableBtnIfInputEmpty(e) {
   const emailBtn = document.getElementById("submit-email");
-  if(e.target.value === ""){
+  if (e.target.value === "") {
       emailBtn.disabled = false;
       removeInvalidMessage();
   } else {
       enableBtnIfEmailValid(e);
   }
 }
 
-function showFalseDoor(){
+function showFalseDoor() {
   handleEvents("SignUp");
   removeClass("#false-door", "hidden");
-  document.getElementById("close-false-door").onclick = function (){
+  document.body.style.overflow = "hidden";
+  const falseDoor = document.getElementById("false-door");
+  window.addEventListener("keydown", function listener(e) {
```

Comment:
the listener function here and the one below added to the false door are different, so removing this one from the false door has no effect. How's this:

https://pastebin.mozilla.org/9088475